crates/sui-core/src/authority.rs

```diff
@@ -1085,26 +1085,44 @@ impl AuthorityState {
         SuiTransactionEffects::try_from(effects, self.module_cache.as_ref())
     }
 
+    /// The object ID for gas can be any object ID, even for an uncreated object
```

Comment:
Does this comment still make sense? This function doesn't accept a gas object ID.

Reply:
Nope, sorry, was from an intermediate version of this PR